test/phoenix/integration/long_poll_channels_test.exs

```diff
@@ -159,7 +159,13 @@ defmodule Phoenix.Integration.LongPollChannelsTest do
   """
   def poll(method, path, vsn, params, json \\ nil, headers \\ %{}) do
     {serializer, json} = serializer(vsn, json)
-    headers = Map.merge(%{"content-type" => "application/json"}, headers)
+    headers =
+      if is_list(json) do
+        Map.merge(%{"content-type" => "application/ndjson"}, headers)
```

Comment:
According to the spec this should be `application/x-ndjson`
https://github.com/ndjson/ndjson-spec#33-mediatype-and-file-extensions